HiP-DataStore.Model/Entity/ReviewComment.cs

```diff
@@ -0,0 +1,35 @@
+﻿using PaderbornUniversity.SILab.Hip.DataStore.Model.Rest;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Entity
+{
+    public class ReviewComment : ContentBase
+    {
+        public string Text { get; set; }
+
+        public bool Approved { get; set; }
+
+        public int ReviewId { get; set; }
```

Comment:
Do we need the connection from Reviews to ReviewComments in both directions? The Reviews already have list of the id's of the ReviewComments, this should be enough to model everything